libqtile/backend/wayland/core.py

```diff
@@ -119,6 +119,7 @@ def __init__(self):
         self.keyboards: List[keyboard.Keyboard] = []
         self.grabbed_keys: List[Tuple[int, int]] = []
         self.grabbed_buttons: List[Tuple[int, int]] = []
+        self.multilang_mapping = MultilanguageMapping()
```

Comment:
Please could you rename this `self._multilang_mapping` as it's only used by this class.